### PR TITLE
Create splash page for web

### LINKED ISSUE #88: Create splash page
The web browser view needs a splash page for external people to learn about the application and for us to show our datasets. Lyndsay is working on a high fidelity mockup of the design that we just need to implement

### PR DESCRIPTION
**Pull request by aswallace**
### Description
This adds the splash page based on [the Figma prototype](https://www.figma.com/design/oNQvOAK11YINcbPvNmkgwO/Public-File-Explorer?node-id=0-1&t=5uvTMOVmFGcVDUVx-0). Currently has placeholders for the carousel images/text and the walkthrough video, as well as empty routes to the datasets and learn pages.
Also adds the global header (web only) with conditional menu items
Closes #88 

### Changes
- Added the router per closed PR #100 
- Added SVG loading to webpack, mainly for the logo, which did not work with the current SVG wrapper 
- Updated styling variables: Roboto --> Open Sans, and new global colors. Have not removed the old variables yet as the app itself still needs to be updated to use the new ones

### Screenshots
Figma design
image

Implementation
image

### CODE REVIEW

File: packages/core/styles/global.css [hunk 1]
```diff
@@ -28,13 +29,22 @@ body {
...
+    --borders-light-grey: #575859;
+    /* Old theme */
     --dark-grey: #313131;
     --medium-grey: #4B4B4B;
     --light-grey: #BFBFBF;
     --white: #ffffff;
     --red: #c23030;
 
     /* DEFAULT THEME (dark theme) */
+    /* To do: Update greys to new theme, check components for consistency */
```
Comment (SeanLeRoy) [inline packages/core/styles/global.css:47]: Glad to see this being kept maintained!

File: packages/web/src/components/Header/Help.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+import styles from "./Help.module.css";
+
+interface Props {
+    path: string;
+}
+
+// Banner for the splash page
+export default function HelpMenu(props: Props) {
+    const isApp: boolean = props.path == "/app";
```
Comment (SeanLeRoy) [inline packages/web/src/components/Header/Help.tsx:14]: (nit) `isAppPage` or `isAppRoute` would be my slight preference to be slightly more specific

File: packages/web/src/components/Header/Help.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            text="Help"
+            menuProps={{
+                className: styles.helpMenu,
+                directionalHint: DirectionalHint.bottomAutoEdge,
+                shouldFocusOnMount: true,
+                items: helpMenuOptions,
+                styles: {
+                    container: {
+                        filter: "drop-shadow(black 100px 14px 18px)",
+                        "background-color": "green !important",
```
Comment (SeanLeRoy) [inline packages/web/src/components/Header/Help.tsx:80]: Is this still being applied? I didn't test this yet, but I didn't see green in the screenshots in your description. Also, if at all possible controlling this kind of style using classNames + css modules would be preferred.

File: packages/web/src/components/Header/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import classNames from "classnames";
+import { DefaultButton } from "@fluentui/react";
+import * as React from "react";
+import { Link, useLocation } from "react-router-dom";
+import styles from "./Header.module.css";
+import HelpMenu from "./Help";
+import { APPLICATION_NAME } from "../../constants";
+import AICSLogo from "../../../assets/AICS-logo-and-name.svg";
```
Comment (SeanLeRoy) [inline packages/web/src/components/Header/index.tsx:8]: Imports in our JS repos are ordered like so:
```
import { DefaultButton } from "@fluentui/react";
import classNames from "classnames";
import * as React from "react";
import { Link, useLocation } from "react-router-dom";

import HelpMenu from "./Help";
import { APPLICATION_NAME } from "../../constants";
import AICSLogo from "../../../assets/AICS-logo-and-name.svg";

import styles from "./Header.module.css";
```
AKA
```
<alphabetical third party imports>

<local imports ordered by how far away they are from this package (like `../../` vs `../../../../..`)>

<style sheet if applicable>
```

File: packages/web/src/index.tsx [hunk 5]
```diff
@@ -13,9 +14,35 @@ import FmsFileExplorer from "../../core/App";
...
+            {
+                path: "app",
+                element: <FmsFileExplorer />,
+            },
+            {
+                path: "datasets",
+                element: <></>, // Datasets placeholder
+            },
+            {
+                path: "learn",
```
Comment (SeanLeRoy) [inline packages/web/src/index.tsx:39]: For the initial release lets exclude "Learn" as an option. We won't have enough concrete examples to put in there in time

**Review by SeanLeRoy — APPROVED** [review]
Awesome work Anya!

File: packages/web/src/components/Header/Help.tsx (hunk 3 above)
Comment (aswallace) [reply packages/web/src/components/Header/Help.tsx:80]: Oops yeah! this was just me testing things out, forgot to remove

File: packages/web/src/index.tsx (hunk 5 above)
Comment (aswallace) [reply packages/web/src/index.tsx]: Removed in [16ec93a](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/107/commits/16ec93a7118115877bbf37cba30987c8fd5483d9)

File: packages/web/src/components/Header/index.tsx (hunk 4 above)
Comment (aswallace) [reply packages/web/src/components/Header/index.tsx:8]: Fixed in [cadb6d6](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/107/commits/cadb6d6ce41eb74806b71dbfc7747b5b3f533c33)

**Pull request merged**